api/v1/models/CheckIn.js

```diff
@@ -0,0 +1,29 @@
+var _ = require('lodash');
+var checkin = require('../utils/check_in');
+var CheckIt = require('checkit');
+
+
+var Model = require('./Model');
+var CheckIn =  Model.extend({
+    tableName: 'checkins',
+    idAttribute: 'id',
+    validations: {
+        userId: ['required', 'integer'],
+        checkedIn: ['required', 'boolean'],
```

Comment:
Why do we have this parameter if a user can only have one check-in? The existence of a check-in should replace this value implicitly.